ckan/lib/navl/dictization_functions.py

```diff
@@ -37,6 +37,9 @@ def __str__(self):
             return repr(self.error)
         return ''
 
+    def __unicode__(self):
+        return unicode(str(self))
```

Comment:
I tried this change but I get the same behaviour before and after when using `print myerror`. How about we return something other than `''` when self.error isn't set? `super(DictizationError, self).__str__()` is one option.